enterprise/models.py

```diff
@@ -118,6 +118,13 @@ class Meta:
         )
     )
 
+    contact_email = models.EmailField(
```

Comment:
@pomegranited I'm having trouble figuring out where `EnterpriseContactUser` came from? It seems like @mattdrayer was talking about potentially adding a `contact_user_id` FK to `EnterpriseCustomerContactInfo` sometime in the future with his last comment. Let me know if I'm getting it wrong, though. :)

I can understand just creating an `EnterpriseCustomerContactInfo` model with a 1-1 relationship with `EnterpriseCustomer`, and giving it only one other field in this PR for now: `learner_contact_email`. Would this make sense @mattdrayer @pomegranited?

It seems like for this particular purpose, we need a single optional email field that represents a general point of contact for the `EnterpriseCustomer` that learners can use - based on the requirements of the message in ENT-216:
![](https://i.imgur.com/JrMZwfb.png)